docs/references/chrome-extension/sync-host.mdx

```diff
@@ -91,6 +91,36 @@ Clerk allows you to sync the authentication state from your web app to your Chro
   }
   ```
 
+  ### Hide unsupported authentication methods
+
+  When using the Sync Host feature you may want to enable authentication methods on the web application that are unsupported in the extension. The Dashboard settings affect anything using the Clerk application, so those methods would show up in your Chrome Extension's sign up and sign in components. Below you will find options to hide unsupported methods.
+
+  <CodeBlockTabs options={["Sign-up", "Sign-in"]}>
+    ```tsx
+    <SignUp
+      routing="virtual"
+      appearance={{
+        elements: {
+          socialButtons: 'plasmo-hidden',
```

Comment:
Not a blocker, but since there is a wrapping element around socialButtons, the parent flex containers gap is causing some extra space above the form fields.

![Screenshot 2024-12-06 at 3 56 15 PM](https://github.com/user-attachments/assets/69b992c0-475e-45cd-9f23-b8adc939a861)

Reply:
Possible solution:

```
appearance: {
  elements: {
    main: {
      '& > div:has(.cl-socialButtons)': {
        display: 'none',
      },
    },
    dividerRow: {
      display: 'none',
    },
  },
},
```

Reply:
opened PR to add a descriptor for you to target actually `socialButtonsRoot` https://github.com/clerk/javascript/pull/4729